ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -362,6 +364,15 @@ object Config {
             "Enable the development version of the Daml-LF language. Highly unstable. Should not be used in production."
           )
 
+        // TODO append-only: remove after removing support for the current (mutating) schema
+        opt[Unit]("index-append-only-schema-unsafe")
+          .optional()
+          .hidden()
+          .text(
+            s"Use the experimental append-only index database. Highly unstable. Should not be used in production."
```

Comment:
Can you please adapt the description? This flag is now enabling parallel ingestion as well, and will be gradually non experimental.